consensus/state_processing/src/common/get_attestation_participation.rs

```diff
@@ -32,9 +33,34 @@ pub fn get_attestation_participation_flag_indices<E: EthSpec>(
     let is_matching_source = data.source == justified_checkpoint;
     let is_matching_target = is_matching_source
         && data.target.root == *state.get_block_root_at_epoch(data.target.epoch)?;
-    let is_matching_head =
+
+    let is_matching_blockroot =
```

Comment:
Honestly i feel this code would be much easier to check if you just assign a variable to `head_root_matches` like the spec does. The spec may over-assign variables but I don't think that matters terribly for performance but it's just much easier to compare to the spec then.